src/components/report/candidate.js

```diff
@@ -26,6 +31,7 @@ export default function CandidateReport() {
   return (
     <section className={style.container}>
       <ArchetypeHeading />
+      {showFeedback && <Feedback />}
```

Comment:
I think we might need to also disable feedback if the locale isn't en-us or we need to get translations for the new text. I'd ask Josh what he wants to do

Reply:
The graphql request to xavier for the survey contains the locale param. The product team can add translation through contentful if needed